batch/gauge_error_analysis.py

```diff
@@ -0,0 +1,410 @@
+"""
+Script to read gauge data and calculate errors relative to the baseline run for the batch runs
+"""
+# ============================================================================
+#      Copyright (C) 2013 Kyle Mandli <kyle@ices.utexas.edu>
+#
+#          Distributed under the terms of the MIT license
+#                http://www.opensource.org/licenses/
+#
+# (post process scripts by Akshay Sriapda <as4928@columbia.edu>, 2017)
+# ============================================================================
+
```

Comment:
You actually can omit this in lieu of the license document in the root directory.  Feel free to update that as well with your name.